pom.xml

```diff
@@ -51,10 +51,38 @@
         <jacoco-listeners.version>3.5</jacoco-listeners.version>
 
     </properties>
+	
+	<repositories>
+             <repository>
+                    <snapshots>
+                          <enabled>false</enabled>
+                    </snapshots>
+                    <id>central</id>
+                    <name>Central Repository</name>
+                    <url>https://repo.maven.apache.org/maven2</url>
+             </repository>
+       </repositories>
+       <pluginRepositories>
+             <pluginRepository>
+                    <releases>
+                          <updatePolicy>never</updatePolicy>
+                    </releases>
+                    <snapshots>
+                          <enabled>false</enabled>
+                    </snapshots>
+                    <id>central</id>
+                    <name>Central Repository</name>
+                    <url>https://repo.maven.apache.org/maven2</url>
+             </pluginRepository>
+       </pluginRepositories>
+
 
     <dependencyManagement>
         <!-- Import the maven Spring IO Platform Bill Of Materials (BOM) -->
+
+
```

Comment:
Could you please remove these extra lines?

Reply:
removed and updated in my repository